builds/running-entitled-builds.adoc

```diff
@@ -10,16 +10,10 @@ include::modules/builds-create-imagestreamtag.adoc[leveloffset=+1]
 
 include::modules/builds-source-secrets-entitlements.adoc[leveloffset=+1]
 
-There are two paths to pulling in the base RHEL image:
-
-* Add the pull secret to registry.redhat.io to your project.
-* Create an imagestream in the `openshift` namespace for the RHEL-based
-image. This makes the imagestream available across the cluster.
+To pull in the base RHEL image, create an imagestream in the `openshift` namespace for the RHEL-based image. This makes the imagestream available across the cluster.
 
 == Running builds with Subscription Manager
```

Comment:
Right now, the "Running builds with Subscription Manager" includes only one subtopic, "Docker builds using Subscription Manager." Does this seem right to you? If so, I'll promote the content up from the subsection to the section.